src/main/index.js

```diff
@@ -47,7 +49,81 @@ function runApp() {
           browserWindow.webContents.selectAll()
         }
       }
-    ]
+    ],
+    // only show the copy link entry for external links and the /playlist, /channel and /watch in-app URLs
+    // the /playlist, /channel and /watch in-app URLs get transformed to their equivalent YouTube URLs
+    append: (defaultActions, parameters, browserWindow) => {
+      let visible = false
+      const isInAppUrl = parameters.linkURL.split('#')[0] === browserWindow.webContents.getURL().split('#')[0]
```

Comment:
Might want to make parameters.linkURL.split('#') into a variable (you seem to be splitting it multiple times

Reply:
Whoops 🙈, sorry about that, added the variable now.